packages/cdk/utils/src/replace-units/replace-units.ts

```diff
@@ -0,0 +1,114 @@
+import { Nullable } from '@angular-ru/cdk/typings';
+
+import { isNil } from '../is-nil';
+import { isNotNil } from '../is-not-nil';
+import { DEFAULT_UNITS_MAP } from './properties';
+import { UnitsMap } from './types';
+
+const REGEXP_NUMBER: string = `[0-9]+([\.][0-9]+)?`;
+
+export function replaceUnits(text: string, unitsMap?: Nullable<UnitsMap>): string {
+    const unitsMapPrepared: UnitsMap = unitsMap ?? DEFAULT_UNITS_MAP;
```

Comment:
and `unitsMapPrepared` -> `unitsMap`
```suggestion
export function replaceUnits(text: string, unitsMap: UnitsMap = DEFAULT_UNITS_MAP): string {
```

Reply:
moved "DEFAULT_UNITS_MAP" to the publiс and made the parameter "unitsMap" not "Nullable" to make it more explicit
```
import { DEFAULT_UNITS_MAP, replaceUnits, UnitsMap } from '@angular-ru/cdk/utils';
expect(replaceUnits('1h', DEFAULT_UNITS_MAP)).toBe('3600');
```